scripts/interactions.js

```diff
@@ -290,8 +290,10 @@ damasflow_ondrop = function ( e )
 			return b.length - a.length;
 		});
 		console.log(workdir);
-		for(var w=0;w<workdir.length;w++)
+		var oldPath=path;
```

Comment:
oldPath is defined because you overwrite the path variable with the stripped value, but you still need the original value. Keeping path, and defining a new value for the stripped is a way easier